src/app/(app)/globals.css

```diff
@@ -2,82 +2,81 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url('../../components/client/slider/productSlider.css');
+
 body {
-    margin: 0;
-    background-color: white;
-    color: #0f0f0f;
+  @apply m-0 bg-[#fbfbfb] text-[#0f0f0f] dark:bg-[#0f0f0f] dark:text-[#fbfbfb];
```

Comment:
you delete     margin: 0;
background-color: white;
color: #0f0f0f; why?

Reply:
Fixed